extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -128,6 +129,42 @@ export const LIFECYCLE_STAGES = {
 };
 
 
+/**
+ * Utility function that ensures any error thrown is handled by optional
+ * onError handler (if none provided or handler throws, error is swallowed).
+ * @param {!Function} fn to protect
+ * @param {T=} opt_this An optional object to use as the 'this' object
+ *    when calling the function.  If not provided, undefined is bound as this
+ *    when calling function.
+ * @param {function(this:T, !Error, ...*):?=} opt_onError function given error
+ *    and arguments provided to function call.
+ * @return {!Function} protected function
+ * @template T
+ * @visibleForTesting
+ */
+export function protectFunctionWrapper(
+    fn, opt_this = undefined, opt_onError = undefined) {
```

Comment:
[Do not prefix optional parameter names with `opt_`.](https://google.github.io/styleguide/jsguide.html#features-functions-default-parameters)